jsonexample/lib/built_value/built_complex_object.dart

```diff
@@ -23,14 +23,19 @@ abstract class BuiltComplexObject
   @nullable
   double get aDouble;
 
+  @nullable
```

Comment:
This makes me sad, but I know you're doing it for a reason.

Reply:
Yeah, I did the other two first, so it's been changed to match. I'm annoyed that I didn't catch it earlier, but that's why one writes tests, I guess. 😄